server/index.js

```diff
@@ -1,42 +1,114 @@
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
-const port = 3001;
-const db = require('./../database/index.js')
 const cors = require('cors');
+const db = require('./../database/mysql.js');
+const port = 3001;
 
 app.use(cors());
 app.use('/', express.static(__dirname + '/./../client/dist'));
 app.use('/:id', express.static(__dirname + '/./../client/dist'));
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }))
+app.use(bodyParser.urlencoded({ extended: true }));
 
+app.listen(port, () => console.log(`Example app listening on port ${port}!`));
 
-app.listen(port, () => console.log(`Example app listening on port ${port}!`))
+// CRUD operations for "houses" endpoint
+app.get('/houses/:id', (req, res) => {
+  const { id } = req.params;
+  db.select('street','city','state','zipcode','description')
+    .from('houses')
+    .where('id', id)
+    .then((house) => {
+      console.log('SUCCESS Here the House data: ', house[0]);
+      (house.length < 1) ? res.sendStatus(404) : res.status(200).json(house);
+    })
+    .catch((error) => {
+      console.error('unable to perform query in server', error);
+      res.status(404).send('Invalid id in houses route');
+    });
+});
 
+app.post('/houses/:id', (req, res) => {
+  const { body } = req;
+  db('houses')
+    .insert(body, ['id'])
+    .then((response) => {
+      console.log('Found the response', response);
+      res.sendStatus(201);
+    })
+    .catch((error) => {
+      console.error('Error adding new House record', error);
+      res.sendStatus(404).end();
+    });
+});
 
-app.get('/houses/:id', (req, res) => {
-  db.House.findAll(
-    {where: {id: req.params.id}}
-  ).then(data => {
-    res.status(200).send(data);
-  })
-  .catch(err => {
-    console.error(err);
-    res.status(404).send()
-  });
+app.put('/houses/:id', (req, res) => {
+  const { id } = req.params;
+  const { body } = req;
+  console.log('here is the id', id);
+  db('houses')
+    .update(body, ['id'])
+    .where('id', id)
+    .then((response) => {
+      console.log('Updated changes', response);
+      res.sendStatus(202);
+    })
+    .catch((error) => {
+      console.error('Unsuccessful update', error);
+      res.status(404).send(error);
+    });
 });
 
+// Uses REPLACE to not break UI
+app.delete('/houses/:id', (req, res) => {
+  const { id } = req.params;
+  const params = [id, '', '', '', 'Deleted House Listing', '00000', 0];
+  const qs = `REPLACE INTO houses(id, street, city, state, description, zipcode, price) \n
```

Comment:
Think you can simply do a delete.  I noticed on testing they don't ask us to test with delete just want us to be able to write it.